documentation/docs/libraries/java/api_reference.md

```diff
@@ -1,11 +1,1096 @@
+
+Creates a new `TransactionPayloadBuilder`.
+
+#### withEssence(essence): TransactionPayloadBuilder
+
+Adds an essence to a `TransactionPayloadBuilder`.
+
+| Param | Type       | Description |
+| ----- | ---------- | ----------- |
+| essence | `Essence` | index bytes |
+
+#### withUnlockBlocks(unlock_blocks): TransactionPayloadBuilder
+
+Adds unlock blocks to a `TransactionPayloadBuilder`.
+
+| Param | Type       | Description |
+| ----- | ---------- | ----------- |
+| unlock_blocks | `UnlockBlocks` | index bytes |
+
+#### finish(): TransactionPayload
+
+Finishes a `TransactionPayloadBuilder` into a `TransactionPayload`.
+
+### TreasuryPayload
+
+#### new(input, output): TreasuryPayload
+
+Creates a new `TreasuryPayload`.
+
+| Param | Type       | Description |
+| ----- | ---------- | ----------- |
+| input | `TreasuryInput` | The input information of this payload |
+| output | `TreasuryOutput` | the output information of this payload |
+
+#### input(): TreasuryInput
+
+Return the TreasuryInput
+
+#### output(): TreasuryOutput
+
+Returns the TreasuryOutput
+
+### MilestonePayload
+
+#### essence(): MilestonePayloadEssence
+
+Returns the essence of a `MilestonePayload`.
+
+#### signatures(): MilestoneSignature[]
+
+Returns the signatures of a `MilestonePayload`.
+
+#### id(): long
+
+Computes the identifier of a `MilestonePayload`.
+
+#### validate(applicable_public_keys, min_threshold): void
+
+Semantically validate a `MilestonePayload`. Throws an error if the milestone is considered invalid.
+
+| Param | Type       | Description |
+| ----- | ---------- | ----------- |
+| applicable_public_keys | `String[]` | hex encoded list of public keys used by the milestone |
+| min_threshold | `long` | Minimum amount of signatures we need to verify on either side |
+
+### ReceiptPayload
+
+#### from(migrated_at, last, funds, transaction): void
+
+Creates a new `ReceiptPayload`.
+
+| Param | Type       | Description |
+| ----- | ---------- | ----------- |
+| migrated_at | `long` | The milestone index at which the funds of a `ReceiptPayload` were migrated |
+| last | `boolean` | whether a `ReceiptPayload` is the final one for a given migrated at index. |
+| funds | `MigratedFundsEntry[]` | The funds which are migrated  |
+| transaction | `MessagePayload` | The `TreasuryTransaction` used to fund the funds. Must be a Treasury type  |
+
+#### migratedAt(): long
+
+Returns the milestone index at which the funds of a `ReceiptPayload` were migrated at in the legacy network.
+
+#### last(): boolean
+
+Returns whether a `ReceiptPayload` is the final one for a given migrated at index.
+
+#### funds(): MigratedFundsEntry[]
+
+The funds which were migrated with a `ReceiptPayload`.
+
+#### amount(): long
+
+Returns the sum of all `MigratedFundsEntry` items within a `ReceiptPayload`.
+
+#### transaction(): TreasuryPayload
+
+The `TreasuryTransaction` used to fund the funds of a `ReceiptPayload`.
+
+### Essence
+
+#### getAsRegular(): Optional<RegularEssence>
```

Comment:
@Thoralf-M , @kwek20 The Docusaurus build error comes from these tags.  Docusuaurus interprets them as HTML tags which are not closed.  There's 2 options to work around this:

1. Close the "tag": `<RegularEssence/>`
2. Use &lt; and &gt; to replace < and >: `&lt;RegularEssence&gt;`

Reply:
I will do option 2. One just looks really weird :)